fix(css): Clarify `top` property for sticky elements (#30612)
mdn/content: files/en-us/web/css/top/index.md

- When `position` is set to `sticky`, the `top` property is used to compute the sticky-constraint rectangle.
- When `position` is set to `static`, the `top` property has _no effect_.

> Consider the situation when both `top` and {{cssxref("bottom")}} values are specified. In this case:

Reviewer: ```suggestion
When both `top` and {{cssxref("bottom")}} values are specified, there are three different cases:
```
